CHANGELOG.md

```diff
@@ -10,18 +10,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Added this very changelog - [#71](https://github.com/DigitalExcellence/dex-backend/issues/71)
-- Setup basic unit test framework to ensure that the core functionality of the application works [#65] (https://github.com/DigitalExcellence/dex-backend/issues/65)
-- Added example unittests for other contributors [#74] (https://github.com/DigitalExcellence/dex-backend/issues/74)
+- Setup basic unit test framework to ensure that the core functionality of the application works [#65](https://github.com/DigitalExcellence/dex-backend/issues/65)
+- Added example unittests for other contributors [#74](https://github.com/DigitalExcellence/dex-backend/issues/74)
 - You can now authenticate swagger with the Identity server - [#101](https://github.com/DigitalExcellence/dex-backend/issues/101)
 - Added issue & pull request templates (bug & report) - [#11](https://github.com/DigitalExcellence/dex-backend/issues/11)
 - Changed errors to be compliant with RFC 7807, with guids for easy error searching - [#80](https://github.com/DigitalExcellence/dex-backend/issues/80)
 - Added highlighted filter to search endpoint - [#57](https://github.com/DigitalExcellence/dex-backend/issues/57)
-- Setup basic unit test framework to ensure that the core functionality of the application works [#65] (https://github.com/DigitalExcellence/dex-backend/issues/65)
-- Added example unittests for other contributors [#74] (https://github.com/DigitalExcellence/dex-backend/issues/74)
+- Setup basic unit test framework to ensure that the core functionality of the application works [#65](https://github.com/DigitalExcellence/dex-backend/issues/65)
 - Added roles and authorization validation. - [#107](https://github.com/DigitalExcellence/dex-backend/issues/107)
 
 ### Changed
 
+- Return Unauthorized instead of Bad Request when not allowed to perform action in controller - [#132](https://github.com/DigitalExcellence/dex-backend/issues/132)
```

Comment:
```suggestion
- Return Forbidden instead of Bad Request when not allowed to perform action in controller - [#132](https://github.com/DigitalExcellence/dex-backend/issues/132)
```